File: LeetCode-Problems/Easy/Shuffle-String.java
//Java
class Solution {
    public String restoreString(String s, int[] indices) {
        String returnedString = "";
        for (int i = 0; i < indices.length; i++){
            for (int j = 0; j < indices.length; j++){
                if (indices[j] == i){
                    returnedString += s.substring(j, j+1);
                }
            }
        }
        return returnedString;
        
    }
}

File: LeetCode-Problems/Easy/Shuffle-String.py
# Python
class Solution(object):
    def restoreString(self, s, indices):
        """
        :type s: str
        :type indices: List[int]
        :rtype: str
        """
        returnedString = ""
        for x in range(0, len(indices)):
            for y in range(0, len(indices)):
                if (indices[y] == x):
                    returnedString += s[y:y+1]
        return returnedString
        
